src/components/android-settings-container.js

```diff
@@ -39,12 +44,18 @@ class AndroidSettingsContainer extends React.Component {
      */
     handleChange = e => {
         e.preventDefault()
-        if (e.target.name === 'valuesTEI') {
-            const valueInput = e.target.value
-            e.target.value > maxValues.valuesTEI
-                ? (e.target.value = maxValues.valuesTEI)
-                : (e.target.value = valueInput)
+
+        let { value } = e.target
```

Comment:
As in #12 this should be `const`

Reply:
Also this will conflict with the other change (the line is different) so you'll need to update this PR once the other is merged